packages/lib-classifier/src/plugins/tasks/experimental/textFromSubject/components/TextFromSubjectTask.stories.js

```diff
@@ -0,0 +1,65 @@
+import asyncStates from '@zooniverse/async-states'
+import React from 'react'
+
+import { MockTask } from '@stories/components'
+import TextFromSubjectTask from './TextFromSubjectTask'
+
+export default {
+  title: 'Tasks / TextFromSubject',
+  component: TextFromSubjectTask,
+  args: {
+    contentLoadingState: asyncStates.success,
+    dark: false,
+    isThereTaskHelp: true,
+    required: false,
+    subjectReadyState: asyncStates.success
+  },
+  argTypes: {
+    contentLoadingState: {
+      control: {
+        type: 'select',
+        options: asyncStates
+      }
+    },
+    subjectReadyState: {
+      control: {
+        type: 'select',
+        options: asyncStates
+      }
+    }
+  },
+  parameters: {
+    viewport: {
+      defaultViewport: 'responsive'
+    }
+  }
+}
+
+export function Default ({ contentLoadingState, dark, isThereTaskHelp, required, subjectReadyState }) {
+  const tasks = {
+    T0: {
+      help: 'Edit the text in the box.',
+      instruction: 'Correct the text',
+      required,
+      taskKey: 'T0',
+      type: 'textFromSubject'
+    }
+  }
+  const subject = {
+    content: 'Herbarium of the University of North Carolina\nSOUTH CAROLINA\nCharleston County\nGnaphalium peregrinum Fern,\nrailroad right-of-way, Johns Island Station on\nCounty Rt. 20 (wes t of Charleston.\nHarry E. Ahles 22002 April 2, 1957\nwith John G. Haesloop\nCollected for the “Flora of the Carolinas"',
+    contentLoadingState,
+    locations: [
+      { 'text/plain': 'http://localhost:8080/subjectContent.txt' }
+    ]
+  }
+
+  return (
+    <MockTask
+      dark={dark}
+      isThereTaskHelp={isThereTaskHelp}
+      subject={subject}
+      subjectReadyState={subjectReadyState}
+      tasks={tasks}
+    />
```

Comment:
The mock task clears the text box when I press Done, but I think that's a bug in `MockTask`. I think it would also be useful to fire an action when the annotation updates, so that we can see changes being saved in the Actions panel.

<img width="612" alt="Screenshot of the task story, showing an empty text input and an empty actions panel." src="https://user-images.githubusercontent.com/59547/159327131-41206707-0bbb-4715-a99b-8a230b4234cd.png">

Reply:
I think adding an action for onChange sounds good. I tried following https://storybook.js.org/docs/react/essentials/actions, including the legacy usage at the bottom, but was unable to implement. Is there an existing story in FEM I could reference?

Reply:
Something like 
```js
{
  argTypes:{
    updateAnnotation: 'hello!'
  }
}
```

should work. It depends how the components props are being passed down from the story to the component itself.

Reply:
I missed that these stories are, in fact, stories for the classifier's `Tasks` component, so won't have that `updateAnnotation` prop, which is only used internally by the text task.

Better task stories are a nice-to-have, but this is outside the scope of this PR.